include/crm/common/options_internal.h

```diff
@@ -21,49 +21,9 @@
 /*
  * Command-line option handling
  *
- * This will all eventually go away as everything is converted to use GOption
  */
```

Comment:
Actually the entire comment is no longer relevant

Reply:
Nuked the whole thing.